sw/device/tests/spi_passthru_test.c

```diff
@@ -71,14 +83,9 @@ bool test_main(void) {
   CHECK_DIF_OK(dif_spi_device_init_handle(
       mmio_region_from_addr(TOP_EARLGREY_SPI_DEVICE_BASE_ADDR), &spid));
 
-  dif_spi_device_config_t dev_cfg = {
-      .clock_polarity = kDifSpiDeviceEdgePositive,
-      .data_phase = kDifSpiDeviceEdgeNegative,
-      .tx_order = kDifSpiDeviceBitOrderMsbToLsb,
-      .rx_order = kDifSpiDeviceBitOrderMsbToLsb,
-      .device_mode = kDifSpiDeviceModePassthrough,
-  };
-  CHECK_DIF_OK(dif_spi_device_configure(&spid, dev_cfg));
+  spi_device_testutils_configure_passthrough(&spid,
+                                             /*filters=*/0,
```

Comment:
Might want to filter everything (from reaching the downstream SPI flash), then selectively enable. Aren't the status register commands currently leaking through to the other end?

Reply:
Good catch.